docs/integration/mongodb.md

```diff
@@ -21,8 +21,13 @@ Sematext MongoDB monitoring agent is an open-source [mongodb monitoring agent](h
 
 ** Generate diagnostics file for Sematext Support **
 
-If you are not seeing some or all MongoDB metrics, see
-[How do I create the diagnostics package](/monitoring/spm-faq/#how-do-i-create-the-diagnostics-package).
+If you are not seeing some or any MongoDB metrics, you can create a "diagnostics dump" and contact us via live chat or email. To create the diagnostics dump just run the following:
+
+```
+sudo spm-client-diagnostics
```

Comment:
@adnanrahic this is really the correct name?  No "mongodb" in the name?

Reply:
Sadly, yes. The Node.js agent had the same name, but it was broken, so while fixing it, I also renamed it into something logical. For this MongoDB agent the script works however. I could rename to the MongoDB diag script but I'd need to release that agent just for that tiny renaming.

I think it's worth it. Give me an hour and I'll make it happen. Then we can keep it like this:

```
sudo spm-mongodb-diagnostics
```

This way we keep a consistent naming across all agents.